SU-HSDO/suhumsci
# feat(STN-58)  foundational css all themes

# READY FOR REVIEW

## JIRA Issue
https://sparkbox.atlassian.net/browse/STN-58

# Summary
Creates the foundational CSS for humsci_basic theme, which will then be utilized in child subthemes. Also, uses ITCSS structure to organize our folder.

- adds normalize, utilities, mixins, variables, flex-grid, and some components from Decanter
- also adds z-index variables
- adds function to get different color maps for each theme
- adds sass-true tests for new function
- adds fonts and colors for the basis of the colorful theme

# Steps to Test
1. See questions I added in the comments.
1. Run `npm run watch` in the humsci_basic folder, add a utility from decanter in `_base.css` to see if it compiles.
2. Run `npm run test` and make sure all tests and linting pass

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
